src/main/java/appeng/container/implementations/ContainerCellWorkbench.java

```diff
@@ -92,11 +92,11 @@ protected int getHeight()
 	@Override
 	protected void setupConfig()
 	{
-		final IInventory cell = this.getUpgradeable().getInventoryByName( "cell" );
+		final IItemHandlerModifiable cell = this.getUpgradeable().getInventoryByName( "cell" );
 		this.addSlotToContainer( new SlotRestrictedInput( SlotRestrictedInput.PlacableItemType.WORKBENCH_CELL, cell, 0, 152, 8, this.getPlayerInv() ) );
 
-		final IInventory inv = this.getUpgradeable().getInventoryByName( "config" );
-		final IInventory upgradeInventory = new Upgrades();
+		final IItemHandlerModifiable inv = this.getUpgradeable().getInventoryByName( "config" );
+		final WrapperLazyItemHandler upgradeInventory = new WrapperLazyItemHandler( new Lazy<>( () -> this.getCellUpgradeInventory() ) );
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1612)